datasets/amazon-last-mile-challenges.yaml

```diff
@@ -16,15 +17,15 @@ Tags:
 
 License: "Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. The material for the Amazon Last Mile Routing Research Challenge is provided under a Creative Commons Attribution-NonCommercial 4.0 International Public License (the “License”). You may not use this material except in compliance with the License. You may obtain a copy of the License at: https://creativecommons.org/licenses/by-nc/4.0/legalcode.txt"
 
-Citation: "Merchan, Daniel; Pachon, Julian; Arora, Jatin; Konduri, Karthik; Winkenbach, Matthias; Parks, Steven; Noszek, Joseph, (2021). 2021 Amazon Last Mile Routing Research Challenge: Training and Evaluation Data. Seattle: Amazon.com."
+Citation:  "Recommended citation: Merchan, Daniel; Pachon, Julian; Arora, Jatin; Konduri, Karthik; Winkenbach, Matthias; Parks, Steven; Noszek, Joseph, (2022). Data for the 2021 Amazon Last Mile Routing Research Challenge. Seattle: Amazon.com. Retrieved from https://registry.opendata.aws/amazon-last-mile-challenges"
```

Comment:
Hi @damercha please check the table at https://github.com/awslabs/open-data-registry#how-are-datasets-added-to-the-registry for the behavior of the citation field. It'll append the contents here to the standard entry, so you can likely cut a lot of this out as it'd be duplicative (e.g., the url).

Reply:
Thanks. I've fixed the citation input accordingly.